### PR TITLE
fix(end_of_quota_nemesis): parse mounts correctly

### PR DESCRIPTION
**Pull request by fruch**
Read all the mount options, and look in them for the expected value

Fixes: #6936

### Testing

- [x] https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fruch/job/longevity-5gb-1h-EndOfQuotaNemesis-aws-test/3
- [x] https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fruch/job/longevity-5gb-1h-EndOfQuotaNemesis-gce-test/3
- [x] https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fruch/job/longevity-5gb-1h-EndOfQuotaNemesis-azure-test/4

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevent to this change (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
with this small change, it pass with all clouds, and with the instance types it had issues with.

File: sdcm/sct_events/group_common_events.py [hunk 1]
```diff
@@ -140,21 +140,25 @@ def ignore_disk_quota_exceeded_errors(node):
             db_event=DatabaseLogEvent.BACKTRACE,
             line="Disk quota exceeded",
             node=node,
+            extra_time_to_expiration=3600,  # one hour
```
Comment (soyacz) [inline sdcm/sct_events/group_common_events.py:143]: why so long?

**Review by soyacz — COMMENTED** [review]
one question only
LGTM

File: sdcm/sct_events/group_common_events.py (hunk 1 above)
Comment (vponomaryov) [reply sdcm/sct_events/group_common_events.py:143]: Because we have log reading lags up to 1 hour?

**Review by vponomaryov — APPROVED** [review]
LGTM

File: sdcm/sct_events/group_common_events.py (hunk 1 above)
Comment (fruch) [reply sdcm/sct_events/group_common_events.py:143]: Yes, but even that wouldn't be enough

There are lags bigger than, it's just a temp patch, that might help on short longevity

**Pull request merged**